tests/ssl-opt.sh

```diff
@@ -403,7 +403,6 @@ detect_required_features() {
 
     case "$CMD_LINE" in
         *server5*|\
-        *server7*|\
```

Comment:
This might have been a typo or copy-paste error for `server6`.

Reply:
Seems more issue rather than server6.
The case "DTLS fragmenting: gnutls client, DTLS 1.2" using server7, with this change, it still failed, I will further check what caused the failure.